annotate.gemspec

```diff
@@ -26,10 +26,10 @@ Gem::Specification.new do |s|
     s.specification_version = 4
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<rake>, [">= 10.4", "< 12.0"])
+      s.add_runtime_dependency(%q<rake>, [">= 10.4", "< 12.1"])
```

Comment:
12 -> 13 should break a ton of stuff if they're using semantic versioning. I tested it with 12, but I can't be sure 13 will work out of the box until I get my hands on that version and can test.